### ISSUE TITLE
RESTIC_PASSWORD_FILE env variables break copy command: Option groups repo2 and from-repo are mutually exclusive

### ISSUE THREAD

**Issue by rsnitsch** [restic/restic#4231]
Output of `restic version`
--------------------------
0.15.1

How did you run restic exactly?
-------------------------------

```
$ set RESTIC_PASSWORD_FILE=password.txt
$ set RESTIC_PASSWORD_FILE2=password.txt
$ restic --from-repo x:\restic\ -r b2:mybucket:restic copy
Fatal: Option groups repo2 and from-repo are mutually exclusive, please specify only one
```

What backend/server/service did you use to store the repository?
----------------------------------------------------------------

Local (source), Backblaze (destination)

Expected behavior
-----------------

Command should work because I am not actually using ``--repo2`` and ``--from-repo`` as the error message claims.

Note that the error only occurs if you set the environment variables RESTIC_PASSWORD_FILE and RESTIC_PASSWORD_FILE2, indicating that this is the source of the mishap.

Actual behavior
---------------

See above. Output is:

`Fatal: Option groups repo2 and from-repo are mutually exclusive, please specify only one`

Steps to reproduce the behavior
-------------------------------

Set RESTIC_PASSWORD_FILE and RESTIC_PASSWORD_FILE2 and then try to copy from one repo to another.

Do you have any idea what may have caused this?
-----------------------------------------------

The internal parsing/handling of the environment variables might be faulty.

Do you have an idea how to solve the issue?
-------------------------------------------

The internal parsing/handling of the environment variables might be faulty.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Yes, great backup software that I use 24/7. Thank you.

**Comment by ProactiveServices** [comment]
It looks like the error is correct: you're using envvars and switches from two different option groups - the "2" and "from" groups.

I believe the "2" suffix for parameters to the `copy` command were replaced with the current pairing of `RESTIC_PASSWORD_FILE` and `RESTIC_PASSWORD_FROM_FILE` along with other `copy` envvars and switches using the "from" syntax. Presumably the old style is still supported for backwards compatibility; they're no longer mentioned in the documentation.

**Comment by rsnitsch** [comment]
I agree with your thesis that probably the two option groups still coexist somewhere in the code and the `RESTIC_PASSWORD_FILE2` is connected to the old ``--repo2`` as opposed to the newer `--from-repo`. Still, I wonder why the code should work that way, because it would be a lot simpler to just treat `--repo2` and `RESTIC_PASSWORD_FILE2` as (deprecated) aliases for their replacements. So I stick to my guess that the commandline/env parsing logic is faulty.

Only using `RESTIC_PASSWORD_FILE2` and the newer `RESTIC_PASSWORD_FROM_FILE` at the same time should throw an error.

But... `RESTIC_PASSWORD_FROM_FILE` does not even exist. It's not mentioned in the docs and it's not recognized by restic either: If I try to set it, restic prompts me for the source password. Also, note that using `RESTIC_PASSWORD_FILE2` does not show a deprecation warning (as opposed to using `--repo2`).
